tox.ini

```diff
@@ -31,3 +30,10 @@ deps=
     -rdoc/requirements.txt
 commands =
     make html
+
+[testenv:quality]
+deps =
+    Django>=1.11,<2.0
```

Comment:
what's the motivation for selecting this django version for running quality tests?

Reply:
I just had to pick one for imports to succeed, and this is the one we plan to be on shortly.  It shouldn't really matter in this context.